src/Mvc/Mvc.Core/test/ModelBinding/Metadata/DefaultModelMetadataTest.cs

```diff
@@ -339,6 +339,56 @@ public void IsBindingRequired_ReturnsFalse_ForTypes(Type modelType)
         Assert.False(isBindingRequired);
     }
 
+    [Fact]
+    public void IsParseableType_ReturnsTrue_ForType()
+    {
+        // Arrange
+        var provider = new EmptyModelMetadataProvider();
+        var detailsProvider = new EmptyCompositeMetadataDetailsProvider();
+
+        var key = ModelMetadataIdentity.ForType(typeof(int));
+        var cache = new DefaultMetadataDetails(key, new ModelAttributes(new object[0], null, null))
+        {
+            BindingMetadata = new BindingMetadata()
+            {
+                IsBindingFromTryParseAllowed = true,
+            },
+        };
+
+        var metadata = new DefaultModelMetadata(provider, detailsProvider, cache);
+
+        // Act
+        var isParseableType = metadata.IsParseableType;
+
+        // Assert
+        Assert.True(isParseableType);
+    }
+
+    [Fact]
+    public void IsParseableType_ReturnsFalse_ForType()
```

Comment:
This test name is misleading, IsParseableType returns false because of the IsBindingFromTryParseAllowed option not because of the type that is used.

Reply:
I think both tests must be renamed. Let me do this.